perma_web/perma/settings/utils/post_processing.py

```diff
@@ -71,3 +71,25 @@ def post_process_settings(settings):
         settings['WORKER_COUNT'] = len([key for key in active.keys() if key.split('@')[0][0] == 'w']) if active else 0
     except TimeoutError:
         pass
+
```

Comment:
Would moving this block to the beginning of `post_process_settings()` make any difference?

Reply:
Good point, it would catch subsequent errors in the file.